examples/KeyWallet/Internal/Pkh2PkhContract.purs

```diff
@@ -52,8 +52,8 @@ runKeyWalletContract_ contract =
           , ctlServerConfig = Nothing
           }
 
-        printLog :: Message -> Aff Unit
-        printLog m = liftEffect $ when (m.level >= Trace) $ do
+        printLog :: LogLevel -> Message -> Aff Unit
+        printLog lgl m = liftEffect $ when (m.level >= lgl) $ do
```

Comment:
I'm still a little bit unclear about where the `logLevel` filtering should be implemented. In every `customLogger` individually, like here? or on a higher level?